src/theory/arith/constraint.cpp

```diff
@@ -1247,6 +1252,19 @@ bool Constraint::allHaveProof(const ConstraintCPVec& b){
   return true;
 }
 
+void Constraint::impliedByIntTighten(ConstraintCP a, bool nowInConflict){
```

Comment:
I suggest including debug messages similar to `impliedByIntHole `.

Reply:
(otherwise LGTM).

Reply:
Sounds good to me. I've done so in d0bbc29fa223f5cea0943b8727573ea1799b285c.